egui_glium/src/lib.rs

```diff
@@ -188,7 +188,7 @@ pub fn input_to_egui(
                 delta.x *= -1.0;
             }
 
-            if input_state.raw.modifiers.ctrl {
+            if input_state.raw.modifiers.command {
```

Comment:
Am I correct in understanding we only want this change to apply on macOS?

If so, I'm guessing the modifier key to match presumably needs to be determined via e.g. `cfg!(target_os = "macos")`?

(Also, as a matter of interest is there a similar issue with the web backend?)

Reply:
By using `modifiers.command` instead of `modifiers.mac_cmd`, the fix should automatically do the right thing, depending on the OS. Sorry, I guess I should have mentioned this in the PR, but I thought this documentation of the constants would technically be sufficient:

```
pub struct Modifiers {
[…]
    /// Either of the control keys are down.
    /// When checking for keyboard shortcuts, consider using [`Self::command`] instead.
    pub ctrl: bool,
[…]
    /// The Mac ⌘ Command key. Should always be set to `false` on other platforms.
    pub mac_cmd: bool,
    /// On Windows and Linux, set this to the same value as `ctrl`.
    /// On Mac, this should be set whenever one of the ⌘ Command keys are down (same as `mac_cmd`).
    /// This is so that egui can, for instance, select all text by checking for `command + A`
    /// and it will work on both Mac and Windows.
    pub command: bool,
```

Reply:
Regarding the web-backend: yes, you are correct. I did not notice it first, since the web backend works for me with a normal pich/zoom gesture (no need for a modifier key). But I tested now by holding CRTL+CMD at the same time (which prevents the hardware zoom from happening), and this reveals that there _would_ we a zoom event if `Command` would be used instead of `CTRL`.

I will update the PR…

Reply:
… should be resolved by <https://github.com/emilk/egui/pull/402/commits/16fd0b9b44a893c10c9fcbd114a876dfe3c92bc6>

Reply:
Ah, thanks for pointing out the docs for `modifiers.command` I wasn't aware that functionality existed. :)

Thanks for investigating & addressing the web backend issue too.

Didn't notice anything obviously broken on Linux/Firefox with the pre-compiled version you uploaded but haven't tested it on Mac myself.

(Only other thing that comes to mind is that we might want to update the UI text in the demo that says to use "Ctrl" to additionally/alternatively mention "Command" key--and could even use the fancy emoji included in egui: https://github.com/emilk/egui/blob/ebb08f87f1adb0b96c6c854bc3a0ed1b7e5cb392/egui_demo_lib/src/apps/demo/font_contents_emoji.rs#L22 :D )

Thanks for your contribution--I imagine Emil will take a look when he has a chance. :)

Reply:
Thanks for hinting at the UI text. <https://github.com/emilk/egui/pull/402/commits/16fd0b9b44a893c10c9fcbd114a876dfe3c92bc6> makes an attempt to address your proposal. Not sure if this is a bit _too much,_ though. Don't hesitate to criticize.

I tested on a Mac with native app, and with wasm in Firefox. It would be great if you (and anyone else) could check the correctness of the UI instructions of the Plot demo on other platforms.